### PR TITLE
update (CORE-124): add referral tracking to signup flow

### PR DESCRIPTION
**Pull request by adplist-david**
### Jira Issue
[[Frontend] Implement Onboarding Invitation Flow](https://adplist.atlassian.net/browse/CORE-124?atlOrigin=eyJpIjoiYjhkOTE2NGI3ODdhNDk5YjkwNzE2NGI3N2MxMjRmZDEiLCJwIjoiaiJ9)

### Context
Goal: Add referral tracking from the onboarding flow to Signup

### Sentry Issue
N. A.

### CODE REVIEW

File: src/Components/AuthConfirm/Signup.js [hunk 1]
```diff
@@ -46,14 +46,15 @@ const Signup = ({ show, proceed, app, user, redirect }) => {
...
+  const [referral] = useGlobal("referral");
 
   /**
    * functions
    */
   const handleSubmit = (params, { setSubmitting, setErrors }) => {
     // signup user
     api
-      .signupService(params)
+      .signupService({ ...params, ...referral })
```
Comment (iamranchojr) [inline src/Components/AuthConfirm/Signup.js:57]: @adplist-david one params needs to be renamed. The referral source, the query param sets it as `src` but the backend expects it as `referral_src`.

**Review by iamranchojr — DISMISSED** [review]
A change needs to be made

File: src/Components/AuthConfirm/Signup.js (hunk 1 above)
Comment (adplist-david) [reply src/Components/AuthConfirm/Signup.js:57]: Yeah.
Done

Comment (adplist-david) [reply src/Components/AuthConfirm/Signup.js:57]: I will share the link for that.
Coming.

Comment (adplist-david) [reply src/Components/AuthConfirm/Signup.js:57]: Is Already done:
`referral_src: param.src`

**Pull request merged**